src/inspector/models/inspectorlistmodel.cpp

```diff
@@ -63,7 +63,8 @@ void InspectorListModel::buildModelsForEmptySelection(const QSet<Ms::ElementType
 {
     static QList<AbstractInspectorModel::InspectorSectionType> persistentSectionList {
         AbstractInspectorModel::InspectorSectionType::SECTION_SCORE_DISPLAY,
-        AbstractInspectorModel::InspectorSectionType::SECTION_SCORE_APPEARANCE
+        AbstractInspectorModel::InspectorSectionType::SECTION_SCORE_APPEARANCE,
+        AbstractInspectorModel::InspectorSectionType::SECTION_NOTATION,
```

Comment:
I encourage you to exclude any changes for inspector from this PR

Reply:
I've only made this change because ElementPopup (see NotationView.qml, line 121) needs the SECTION_NOTATION model to be available at startup